test/e2e/kubelet_perf.go

```diff
@@ -193,6 +196,10 @@ var _ = framework.KubeDescribe("Kubelet [Serial] [Slow]", func() {
 	var rm *framework.ResourceMonitor
 
 	BeforeEach(func() {
+		// Wait until image prepull pod has completed so that they wouldn't
+		// affect the runtime cpu usage. Fail the test if prepulling cannot
+		// finish in time.
+		ExpectNoError(framework.WaitForPodsSuccess(f.Client, api.NamespaceSystem, framework.ImagePullerLabels, imagePrePullingLongTimeout))
```

Comment:
In my version I had 
```
if err := framework.WaitForPodSuccess(f.Client, api.NamespaceSystem, framework.ImagePullerLabels, imagePullGrace); err != nil {
  framework.Failf("Image puller didn't complete in %v, not running resource usage test since the metrics might be adultrated", imagePullGrace)
}
```